tests/unit/jobserver/models/test_outputs.py

```diff
@@ -103,9 +97,16 @@ def test_publishrequest_create_from_files_with_existing_snapshot():
     snapshot2.files.set(files)
 
     with pytest.raises(Snapshot.MultipleObjectsReturned):
-        PublishRequest.create_from_files(
-            files=files, user=UserFactory(), workspace=workspace
-        )
+        PublishRequest.create_from_files(files=files, user=UserFactory())
+
+
+def test_publishrequest_create_from_files_with_multiple_workspaces():
+    user = UserFactory()
+
+    rfiles = ReleaseFileFactory.create_batch(2)
```

Comment:
I take it that the workspace here defaults to a unique thing that's created for each file. It would be easier to understand this test if we explicitly created files with different workspaces, since that's cruicial to the correctness of the test.

Reply:
Yes, by default all our factories create unique items for their FKs, but point taken 👍